src/components/dialogs/searchDialog/PluginsManager.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-dialog
+    v-model="searchEngineState.isPluginManagerOpen"
+    max-width="20rem"
+    scrollable
+  >
+    <v-card>
+      <v-card-title> <v-icon>mdi-toy-brick</v-icon> Plugins manager </v-card-title>
+      <v-card-text>
+        <v-switch
+          v-for="(plugin, key) in searchEngineState.searchPlugins"
+          :key="key"
+          :false-value="false"
```

Comment:
Does this required?

Reply:
`:false-value` by default sets `null`

Reply:
Actually, set `false-value` isn't working, the change event value still null. I believe it's caused by `:value`